src/python/pants/backend/python/rules/pex.py

```diff
@@ -455,7 +460,10 @@ async def two_step_create_pex(two_step_pex_request: TwoStepPexRequest) -> TwoSte
             #  Some of them may affect resolution behavior, but others may be irrelevant.
             #  For now we err on the side of caution.
             additional_args=request.additional_args,
-            description=f"Resolving {', '.join(request.requirements.requirements)}",
+            description=(
+                f"Resolving {pluralize(len(request.requirements.requirements), 'requirement')}: "
+                f"{', '.join(request.requirements.requirements)}"
```

Comment:
NB that this is the TwoStepPex type. It's an implementation detail that we're using two PEXes, so we don't want the default message of "Building __requirements.pex with 20 requirements". The only part that is relevant is the requirements bit.